tools/wasmedge/wasmedger.cpp

```diff
@@ -125,11 +125,11 @@ int main(int Argc, const char *Argv[]) {
     Conf.removeProposal(WasmEdge::Proposal::ReferenceTypes);
   }
   if (PropSIMD.value()) {
-    Conf.addProposal(WasmEdge::Proposal::SIMD);
-  }
-  if (PropAll.value()) {
-    Conf.addProposal(WasmEdge::Proposal::SIMD);
+    Conf.removeProposal(WasmEdge::Proposal::SIMD);
   }
+  /// Left for the future proposals.
```

Comment:
Why do you comment these out?

Reply:
We've turned on the all proposals we implemented now.

Reply:
Why not delete them directly?

Reply:
I think we should keep this for the proposals we will implement and not merge into the official spec in the future?